apps/demo-game/src/reducers/PeriodResultReducer.ts

```diff
@@ -1,65 +1,97 @@
-import { Action } from '@gbl-uzh/platform'
+import {
+  PayloadPeriodResult,
+  PayloadPeriodResultEnd,
+  ResultState,
+} from '@gbl-uzh/platform'
 import { debugLog } from '@gbl-uzh/platform/dist/lib/util'
-import { PrismaClient } from '@prisma/client'
-import { match } from 'ts-pattern'
+import { PeriodFacts, PeriodSegmentFacts } from '@graphql/index'
+import { produce } from 'immer'
 
 const INITIAL_CAPITAL = 10000
 
-export enum ActionTypes {
-  PERIOD_RESULTS_INITIALIZE = 'PERIOD_RESULTS_INITIALIZE',
-  PERIOD_RESULTS_START = 'PERIOD_RESULTS_START',
-  PERIOD_RESULTS_END = 'PERIOD_RESULTS_END',
+type State = {
+  decisions: {
+    bank: boolean
+    bonds: boolean
+    stocks: boolean
+  }
+  assets: {
+    bank: number
+    bonds: number
+    stocks: number
+    totalAssets: number
+  }
 }
 
-type Actions =
-  | Action<ActionTypes.PERIOD_RESULTS_INITIALIZE, any, PrismaClient>
-  | Action<ActionTypes.PERIOD_RESULTS_START, any, PrismaClient>
-  | Action<ActionTypes.PERIOD_RESULTS_END, any, PrismaClient>
+// TODO(JJ): Define somewhere else -> discuss with RS
+export enum PlayerRole {
```

Comment:
yes, probably in a settings/constants file somewhere with some other global things that are user-specific